src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -593,33 +600,61 @@ Profile* CascadiaSettings::_FindMatchingProfile(const Json::Value& profileJson)
 }
 
 // Method Description:
-// - Finds the "default profile settings" if they exist in the users settings,
-//   and applies them to the existing profiles. The "default profile settings"
-//   are settings that should be applied to every profile a user has, with the
-//   option of being overridden by explicit values in the profile. This should
-//   be called _after_ the defaults have been parsed and dynamic profiles have
-//   been generated, but before the other user profiles have been loaded.
+// - Finds the "default profile settings" if they exist in the default settings and
+//   user's settings, then applies them to the existing profiles.
+//   The "default profile settings" are settings that should be applied to
+//   every profile a user has, with the option of being overridden by explicit values
+//   in the profile. This should be called _after_ the defaults have been parsed
+//   and dynamic profiles have been generated, but before the other user profiles
+//   have been loaded.
 // Arguments:
 // - <none>
 // Return Value:
 // - <none>
-void CascadiaSettings::_ApplyDefaultsFromUserSettings()
+void CascadiaSettings::_ApplyDefaultsForProfiles()
 {
     // If `profiles` was an object, then look for the `defaults` object
     // underneath it for the default profile settings.
-    auto defaultSettings{ Json::Value::null };
+    auto primaryDefaultSettings{ Json::Value::null };

```

Comment:
This is implemented in somewhat the wrong order. Right now we...

1. Load defaults/profiles.list
2. Apply defaults/profiles.defaults
3. Apply user/profiles.defaults
4. Load user/profiles.list

when what we really want is to swap 1 and 2.

Reply:
Yeah. This was a lot larger and more complex of a change than we hoped. Removed the code changes and opened #5276